apps/dashboard/src/components/engine/status/status.tsx

```diff
@@ -0,0 +1,31 @@
+import { Flex, Icon, Stack } from "@chakra-ui/react";
+import { FaChartArea } from "react-icons/fa";
+import { Card, Heading } from "tw-components";
+import { ErrorRate } from "./components/ErrorRate";
+import { Healthcheck } from "./components/Healthcheck";
+import { StatusCodes } from "./components/StatusCodes";
+import { TransactionVolume } from "./components/TransactionVolume";
+interface EngineStatusProps {
+  instanceUrl: string;
+}
+
+export const EngineStatus: React.FC<EngineStatusProps> = ({ instanceUrl }) => {
+  return (
+    <>
+      <Card p={16}>
+        <Stack spacing={4}>
+          <Flex gap={2} align="center" pb={-2}>
+            <Icon as={FaChartArea} />
+            <Heading size="title.md">Metrics</Heading>
+          </Flex>
+          <div className="grid grid-cols-1 md:grid-cols-2 lg:grid-cols-3 mt-10 gap-4">
+            <Healthcheck rpc={instanceUrl} />
```

Comment:
`rpc` feels misleading in terms of prop naming here

Reply:
done